frontend/src/metabase/visualizations/visualizations/RowChart/utils/events.ts

```diff
@@ -33,13 +34,17 @@ import { isMetric } from "metabase-lib/types/utils/isa";
 const getMetricColumnData = (
   columns: DatasetColumn[],
   metricDatum: MetricDatum,
+  visualizationSettings: VisualizationSettings,
 ) => {
   return Object.entries(metricDatum).map(([columnName, value]) => {
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     const col = columns.find(column => column.name === columnName)!;
+    const key =
+      getIn(visualizationSettings, ["series_settings", col.name, "title"]) ??
```

Comment:
Shouldn't it be https://github.com/metabase/metabase/blob/61d905cff05cd4049418d8672c2575d90f2a30b8/frontend/src/metabase-lib/queries/utils/get-column-key.ts#L14? The logic is much more complicated than just using `col.name`

Reply:
@ranquild sorry, actually the initial variant was correct. We use `getColumnKey` for `column_settings` but for `series_settings` we rely on `seriesKey` and in case of row charts which cannot be combined on dashboards from multiple cards the key is column name